packages/zowe-explorer/package.json

```
@@ -50,7 +50,7 @@
         {
           "id": "zowezosconsole",
           "title": "%zowe.zosconsole%",
-          "icon": ""
+          "icon": "resources/dark/chevron-right.svg"
```

I'm a fellow dark theme enjoyer, but we should make sure to support light themes 😋 

VS Code supports defining an `icon` object with separate `light` and `dark` files. Alternatively, I think we could use a codicon here which is simpler because then we don't need to bundle a new icon file:
```suggestion
          "icon": "$(chevron-right)"
```

---

Thanks for addressing, can we delete the `chevron-right.svg` file from this PR?